react-backend/client/src/App.js

```diff
@@ -33,6 +31,8 @@ import ConfirmPassword from './cb-admin/pages/ConfirmPassword';
 import ForgotPassword from './cb-admin/pages/ForgotPassword';
 import ResetPassword from './cb-admin/pages/ResetPassword';
 import CbDashboard from './cb-admin/pages/Dashboard';
+import CbAdminFeedbackPage from './cb-admin/pages/FeedbackPage';
+import CbSettingsPage from './cb-admin/pages/Settings';
```

Comment:
Should this be called `CbAdminSettingsPage`? I thought this was the naming convention we were going with..

Reply:
Yeah, sure, I'll rename them all in a future PR.